ghost/core/core/server/data/migrations/versions/5.5/2022-07-21-08-56-add-jobs-table.js

```diff
@@ -0,0 +1,11 @@
+const {addTable} = require('../../utils');
+
+module.exports = addTable('jobs', {
+    id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+    name: {type: 'string', maxlength: 191, nullable: false, unique: true},
+    status: {type: 'string', maxlength: 50, nullable: false, defaultTo: 'queued', validations: {isIn: ['started', 'finished', 'failed', 'queued']}},
+    started_at: {type: 'dateTime', nullable: true},
+    finished_at: {type: 'dateTime', nullable: true},
+    created_at: {type: 'dateTime', nullable: false},
+    updated_at: {type: 'dateTime', nullable: false}
```

Comment:
Is there a reason `updated_at` is `nullable: false`? If we first insert the job, I'd argue it hasn't been "updated" yet? I spoke with @ErisDS about this discrepancy on other tables but if we agree, it'd be good to avoid adding another one

Reply:
Oh. It is a very minor detail, but I do like this sort of correctness. It does make sense!

Reply:
The thing with the `updated_at` is that it's [being set by Bookshelf](https://bookshelfjs.org/api.html#Model-instance-hasTimestamps) no matter what during the insert. We could disable this default behavior and rely solely on the user data. This is a slightly bigger change that's not in the scope here :sweat_smile:

Reply:
🤔 I'd still set it to non-nullable to prepare for the lovely day that we nuke Bookshelf 😈

Reply:
Yeah totally!

Reply:
Sorry I didn't get to this before - but what Daniel said :D